csharp/Platform.IO/FileHelpers.cs

```diff
@@ -20,6 +52,22 @@ public static T[] ReadAll<T>(string path)
             return reader.ReadAll<T>();
         }
 
+        /// <summary>
+        /// <para>Reads and returns <typeparamref name="T"/> struct from <paramref name="path"/>.</para>
+        /// <para>Считывае, и возвращает структуру типа <typeparamref name="T"/> из <paramref name="path"/>.</para>
+        /// </summary>
+        /// <typeparam name="T">
+        /// <para>The structure type.</para>
+        /// <para>Тип являющийся структурой.</para>
+        /// </typeparam>
+        /// <param name="path">
+        /// <para>The path to the file, from which to read the array of <typeparamref name="T"/> structures.</para>
+        /// <para>Путь к файлу, из которого нужно прочитать массив структур типа <typeparamref name="T"/>.</para>
+        /// </param>
+        /// <returns>
+        /// <para>A <typeparamref name="T"/> struct valueif read from <paramref name="path"/> is successfull, otherwise the default struct.</para>
```

Comment:
`valueif read from <paramref name="path"/> is successfull` => `value if read from <paramref name="path"/> is successful`

Reply:
`the default struct`  ⇾ `the default structure value`